pmoves/tools/observability/metrics_specialist.py

```
@@ -0,0 +1,338 @@
+        start = end - timedelta(hours=hours)
+
+        result = self.query_range(metric, start, end, step="1h")
+
+        if not result["data"]["result"]:
+            return {"error": "No data returned for metric"}
+
+        # Extract values for trend analysis
+        values = []
+        for series in result["data"]["result"]:
+            for value in series.get("values", []):
+                timestamp, value = value
```

Ruff (0.15.10)

[warning] Unpacked variable `timestamp` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)